status-im/status-react
# Make sure mobile understands community share link by Desktop

fixes #14560

### Summary

When a community is created on Desktop its share url looks like : `https://join.status.im/c/zQ3shTAten2v9CwyQD1Kc7VXAqNPDcHZAMsfbLHCZEx6nFqk9`

When a community is created on Mobile its share url looks like : 
`https://join.status.im/c/0x025596a7ff87da36860a84b0908191ce60a504afc94aac93c1abd774f182967ce6`

This PR attempts to translate the received community ID ( when you consider the above example )
from 
`zQ3shTAten2v9CwyQD1Kc7VXAqNPDcHZAMsfbLHCZEx6nFqk9`
to 
`0x025596a7ff87da36860a84b0908191ce60a504afc94aac93c1abd774f182967ce6`

So that mobile can access community info from mailserver and it can become usable on mobile.

We should ultimately move communities to only support the shorter key format on mobile as well, 
now that I have the necessary research in place, I will do that in a separate PR.

#### Platforms
- Android
- iOS

##### Functional

- 1-1 chats
- public chats
- group chats
-communities

### Steps to test


- Open Status
- Go to Chat
- Paste share link received from desktop into a chat with anyone
- You should should be able to navigate to the community on tap 


status: ready
